hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/MigrationProperties.java

```diff
@@ -0,0 +1,33 @@
+package com.hedera.mirror.importer.migration;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class MigrationProperties {
+
+    @NotNull
+    private Integer checksum;
```

Comment:
If it's not null then use primitive.

```suggestion
    private int checksum = 1;
```

Reply:
done